### DISCUSSION TITLE
Kafka initContainer OOMKilled: How can I increase its resources?

### DISCUSSION THREAD

**Discussion by mdemierre** [strimzi/strimzi-kafka-operator#6553] [Q&A] [answered]
Hello,

I'm trying to launch our local dev Kafka environment using the Strimzi 0.23 operator. I noticed that as I'm now running it on an M1 Mac with Docker Desktop Kubernetes, the emulation for x64 containers seems to make them use more memory which causes issues with the default resource settings.

Initially I had issues with the operator itself, but those were resolved by editing its deployment to increase the resources. Now I'm stuck with the init container for Kafka brokers being OOMKilled and I can't find how to configure it. I can configure the resources of the Kafka containers, but it doesn't seem to have an effect on their init containers.

Is it possible and if so how?

Note that we're using the 0.23 version because the newer versions don't support Kafka 2.6.1, and this is what our production runs at the moment.

**Comment by scholzj** [comment, marked as answer]
I'm afraid you cann't do it. I never saw it run out of memory, but I also never run it with emulation on Apple M1. The latest version has Arm support and might work better.

**Reply by mdemierre** [reply]
Alright, I was afraid of that. I'll try newer versions, even though it means moving further from our prod environment since lower Kafka versions are not supported.
